contents/handbook/people/team-structure/website-docs.md

```diff
@@ -23,6 +23,18 @@ We’re here to:
 - Own posthog.com
 - Own the docs
 
+## Q3 2022 Goals
+* **Objective**:
+    * Make posthog.com a great resource for new and returning users
+* **Key Results**:
+    * 25% decrease in support tickets per customer
+    * Ensure the top 50 [most visited Docs pages](https://docs.google.com/spreadsheets/d/1awe15l2_Hy8EMcbGeiCLlhv03pRpiqD3PG_7Q_FnESo/edit#gid=420779973) are up to date
+    * Make sure the conversion rate doesn’t fall
```

Comment:
Is there any reason we wouldn't look to improve this rather than maintain?

Reply:
Good q and yes - very consciously.

I think it's critical we care about brand, and adding a ton of CTAs and focussing on conversion in this team will likely harm this. For example, it's well known that removing content and harming transparency often improves conversion. In the long run, this will (in a hard to attribute way) hurt brand searches and word of mouth.

Reply:
I agree with you @jamesefhawkins - @andyvan-ph for context, I was in the meeting when this was discussed - but I think it's reductive to suggest that improve conversion = add a ton of CTAs. 

There are plenty of ways to sensibly improve conversion if this was a focus in future (e.g. better structured information, easy-to-understand pricing, slick payment flow), and I think it would be more productive if we don't push this narrative that 'marketing people (at PostHog) just want to cover the page with CTAs and reduce transparency to improve conversion'. Otherwise we risk dismissing genuinely good ideas. 

(But to clarify, I still agree the key result as it is written!)